content/en/about.md

```diff
@@ -8,6 +8,43 @@ and wider scientific Python community. For more information on our governance
 approach, please see our
 [Governance Document](https://scipy.github.io/devdocs/dev/governance/governance.html).
 
+## Steering Council
+
+The role of the SciPy Steering Council is to ensure, through working with and
+serving the broader SciPy community, the long-term well-being of the project,
+both technically and as a community. The SciPy Steering Council currently
+consists of the following members (in alphabetical order):
+
+* Andrew Nelson
+* Charles Harris
+* Christoph Baumgarten
+* CJ Carey
+* Eric Larson
+* Evgeni Burovski
+* Ilhan Polat
+* Josef Perktold
+* Josh Wilson
+* Matt Haberland
+* Matthew Brett
+* Nikolay Mayorov
+* Paul van Mulbregt
+* Pauli Virtanen (BDFL)
+* Ralf Gommers (Chair)
+* Tyler Reddy (Release manager)
+* Warren Weckesser
+
+Emeritus:
+
+* Anne Archibald
+* Eric Jones
+* Eric Moore
+* Eric Quintero
+* Jaime Fernández del Río
+* Jarrod Millman
+* Robert Kern
+* Stéfan van der Walt
+* Travis Oliphant (Creator of SciPy)
```

Comment:
SciPy was created when Travis (multipack), Eric (Cluster of Workers, GA, and weave), and Pearu (f2py) joined efforts.

This parenthetical may be a bit out of place in this list, because not all founders' names appear here. Perhaps rather mention it in a paragraph about the history of the library?

Reply:
According to [github](https://github.com/scipy/scipy/graphs/contributors?from=2001-01-28&to=2004-06-03&type=c) between Jan 28, 2001 – Jun 3, 2004, Travis, Pearu, and Eric are the only contributors.  That may miss SVN accounts that didn't translate to github.  It also doesn't track the contributions to the code that were made before they were combined.

Reply:
I suggest to list all three as co-creators